auth0/src/main/java/com/auth0/android/authentication/storage/CredentialsManager.kt

```diff
@@ -113,6 +132,9 @@ public class CredentialsManager @VisibleForTesting(otherwise = VisibleForTesting
         if (scope != null) {
             request.addParameter("scope", scope)
         }
+
+        request.addParameters(parameters)
```

Comment:
might make sense to move this line above the "add scope" line, so that particular parameter always has precedence

Reply:
and done